tools/elena/extract_transform.py

```diff
@@ -0,0 +1,188 @@
+    'Instance Segmentation': 'configs/mmdet/instance-seg/instance-seg_tensorrt_static-800x1344.py',
+    'Semantic Segmentation': 'configs/mmseg/segmentation_tensorrt_static-512x512.py',
+    'Oriented Object Detection': 'configs/mmrotate/rotated-detection_tensorrt-fp16_dynamic-320x320-1024x1024.py',
+    'Text Recognition': 'configs/mmocr/text-recognition/text-recognition_tensorrt_static-32x32.py',
+    'Text Detection': 'configs/mmocr/text-detection/text-detection_tensorrt_static-512x512.py',
+    'Restorers': 'configs/mmedit/super-resolution/super-resolution_tensorrt_static-256x256.py'
+}  # yapf: disable
+
+INFO = {
+    'cpu':
+    '''
+using std::string;
+
+void FuseFunc(void* stream, uint8_t* data_in, int src_h, int src_w, const char* format,
+              int resize_h, int resize_w, const char* interpolation, int crop_top, int crop_left,
+              int crop_h, int crop_w, float mean0, float mean1, float mean2, float std0, float std1,
+              float std2, int pad_top, int pad_left, int pad_bottom, int pad_right, int pad_h,
+              int pad_w, float pad_value, float* data_out, int dst_h, int dst_w) {
+  const char* interpolation_ = "nearest";
+  if (strcmp(interpolation, "bilinear") == 0) {
+      interpolation_ = "bilinear";
+  }
+  FuseKernel(resize_h, resize_w, crop_h, crop_w, crop_top, crop_left, mean0, mean1, mean2, std0, std1, std2,
+              pad_h, pad_w, pad_top, pad_left, pad_bottom, pad_right, pad_value, data_in, data_out,
+              src_h, src_w, format, interpolation_);
+}
+
+REGISTER_FUSE_KERNEL(#TAG#_cpu, "#TAG#_cpu",
+                     FuseFunc);
+''',
+    'cuda':
+    '''
+void FuseFunc(void* stream, uint8_t* data_in, int src_h, int src_w, const char* format,
+              int resize_h, int resize_w, const char* interpolation, int crop_top, int crop_left,
+              int crop_h, int crop_w, float mean0, float mean1, float mean2, float std0, float std1,
+              float std2, int pad_top, int pad_left, int pad_bottom, int pad_right, int pad_h,
+              int pad_w, float pad_value, float* data_out, int dst_h, int dst_w) {
+  cudaStream_t stream_ = (cudaStream_t)stream;
+  const char* interpolation_ = "nearest";
+  if (strcmp(interpolation, "bilinear") == 0) {
+    interpolation_ = "bilinear";
+  }
+
+  FuseKernelCU(stream_, resize_h, resize_w, crop_h, crop_w, crop_top, crop_left, mean0, mean1, mean2, std0,
+               std1, std2, pad_h, pad_w, pad_top, pad_left, pad_bottom, pad_right, pad_value, data_in,
+               data_out, dst_h, dst_w, src_h, src_w, format, interpolation_);
+}
+
+REGISTER_FUSE_KERNEL(#TAG#_cuda, "#TAG#_cuda",
+                     FuseFunc);
+'''
+}
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Extract transform.')
+    parser.add_argument('root_path', help='root path to codebase')
+    parser.add_argument('mmmdeploy_path', help='mmdeploy path')
```

Comment:
I think `mmdeploy_path` is not necessary, since it can be de deduced from the path of `extract_transform.py`